docs/fedora-releases-guide/dist-git-onboarding.md

```diff
@@ -146,6 +146,15 @@ to have your package monitored by Upstream Release Monitoring:
 1. make sure there is Fedora mapping for the upstream project in https://release-monitoring.org/projects/
 2. make sure `Monitoring status` on the left side at https://src.fedoraproject.org/rpms/your-package is set to `Monitoring`
 
+:::caution
+
+There could be a variable amount of time between a new upstream tag/release (detected by the release monitoring) and the archive upload.
+
+Packit does 6 retries in 1 min, 2 mins, 4 mins, 8 mins, 16 mins and 32 mins, trying to download the remote archive and after that it raises an `ERROR  Failed to download source from ...`.
```

Comment:
When a download error is detected, the job is automatically retried 6 times over the span of 1 hour and only if the final retry fails the job is marked as failed.